### PR TITLE
Add storage account for LD Location

### PR DESCRIPTION
**Pull request by vijayrajagopalan-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-1901

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: storage-account-ld-location.tf [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+module "storage_account_ld_location" {
+  source                   = "git@github.com:hmcts/cnp-module-storage-account?ref=master"
+  env                      = "${var.env}"
+  storage_account_name     = "${local.ld_location_account_name}"
+  resource_group_name      = "${azurerm_resource_group.rg.name}"
+  location                 = "${var.location}"
+  account_kind             = "BlobStorage"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  access_tier              = "Hot"
```
Comment (shashank-rastogi) [inline storage-account-ld-location.tf:17]: lets keep replication as ZRS for some redundancy

Comment (vijayrajagopalan-hmcts) [reply storage-account-ld-location.tf:17]: Should we consider having ZRS for prod and LRS for lower environments to save cost?

Comment (shashank-rastogi) [reply storage-account-ld-location.tf:17]: yes thats better!!

Comment (vijayrajagopalan-hmcts) [reply storage-account-ld-location.tf]: Have made the changes, pls review and approve.

File: storage-account-ld-location.tf [hunk 2]
```diff
@@ -0,0 +1,66 @@
+locals {
+  product                   = "ldlocation"
+  ld_location_account_name  = "${local.product}${var.env}"
+  container_name            = "lrd-ref-data"
```
Comment (shashank-rastogi) [inline storage-account-ld-location.tf:4]: can you plc change stg account name to rdlocationaat?

Comment (vijayrajagopalan-hmcts) [reply storage-account-ld-location.tf:4]: Change completed.

**Review by shashank-rastogi — APPROVED** [review]

**Review by pksaxena01 — APPROVED** [review]
Approved

**Pull request merged**